app/controllers/admin/vacations_controller.rb

```diff
@@ -0,0 +1,43 @@
+module Admin
+  class Admin::VacationsController < ApplicationController
+    before_action :authenticate_user!
+    before_action :load_vacation, only: [:update, :destroy]
+
+    def index
+      @vacations = Vacation.where(status: 'pending')
```

Comment:
sorry, missed during the first review. If we use `enum` we have built in methods like `pending`, `accepted`, etc. (depending on values of course).

http://api.rubyonrails.org/classes/ActiveRecord/Enum.html